Snippets/Sqs/Sqs_All/QueueCreation/QueueCreationUtils.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace SqsAll.QueueCreation
+{
+    using System;
+    using System.Threading.Tasks;
+    using Amazon.SQS;
+    using Amazon.SQS.Model;
+    using Sqs_All;
+
+    #region sqs-create-queues
+
+    public static class QueueCreationUtils
+    {
+        static TimeSpan DefaultTimeToLive = TimeSpan.FromDays(4);
+
+        public static async Task CreateQueue(string queueName, TimeSpan? maxTimeToLive = null)
+        {
+            try
+            {
+                using (var client = ClientFactory.CreateSqsClient())
+                {
+                    var sqsRequest = new CreateQueueRequest
+                    {
+                        QueueName = QueueNameHelper.GetSqsQueueName(queueName)
+                    };
+                    var value = Convert.ToInt32((maxTimeToLive ?? DefaultTimeToLive).TotalSeconds).ToString();
+                    sqsRequest.Attributes.Add(QueueAttributeName.MessageRetentionPeriod, value);
+                    await client.CreateQueueAsync(sqsRequest)
+                        .ConfigureAwait(false);
+                }
+            }
+            catch (QueueNameExistsException)
```

Comment:
When a queue is created with the same name as another queue that was deleted less than 60sec ago, `CreateQueueAsync` throws `QueueDeletedRecentlyException` - the user would just need to wait 60 sec and try again. Do we want to add this handling here?